doc/source/whatsnew/v1.2.0.rst

```diff
@@ -292,6 +292,7 @@ Other enhancements
 - Improved error reporting for subsetting columns of a :class:`.DataFrameGroupBy` with ``axis=1`` (:issue:`37725`)
 - Implement method ``cross`` for :meth:`DataFrame.merge` and :meth:`DataFrame.join` (:issue:`5401`)
 - When :func:`read_csv/sas/json` are called with ``chuncksize``/``iterator`` they can be used in a ``with`` statement as they return context-managers (:issue:`38225`)
+- Added ``end`` and ``end_day`` options for ``origin`` in :meth:`DataFrame.resample` (:issue:`37804`)
```

Comment:
Not sure this will make it in for 1.2 - might need to move to 1.3

Reply:
@jreback keep this in 1.2 or remove to 1.3 whatsnew?

Reply:
Agreed that this should target 1.3

Reply:
done